drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -670,6 +692,35 @@ static int stm32_clock_control_init(const struct device *dev)
 
 #endif /* CONFIG_CPU_CORTEX_M4 */
 
+#if defined(CONFIG_CLOCK_STM32_PLL2_ENABLE)
+	/* Initialize PLL 2 */
+	r = get_vco_input_range(CONFIG_CLOCK_STM32_PLL2_M_DIVISOR, &vco_input_range);
+	if (r < 0) {
+		return r;
+	}
+	vco_output_range = get_vco_output_range(vco_input_range);
+	LL_RCC_PLL2FRACN_Disable();
+	LL_RCC_PLL2_SetM(CONFIG_CLOCK_STM32_PLL2_M_DIVISOR);
+	LL_RCC_PLL2_SetN(CONFIG_CLOCK_STM32_PLL2_N_MULTIPLIER);
+	LL_RCC_PLL2_SetVCOInputRange(vco_input_range);
+	LL_RCC_PLL2_SetVCOOutputRange(vco_output_range);
+#if defined(CONFIG_CLOCK_STM32_PLL2_P_ENABLE)
+	LL_RCC_PLL2P_Enable();
+	LL_RCC_PLL2_SetP(CONFIG_CLOCK_STM32_PLL2_P_DIVISOR);
+#endif /* CONFIG_CLOCK_STM32_PLL2_P_ENABLE */
+#if defined(CONFIG_CLOCK_STM32_PLL2_Q_ENABLE)
+	LL_RCC_PLL2Q_Enable();
+	LL_RCC_PLL2_SetQ(CONFIG_CLOCK_STM32_PLL2_Q_DIVISOR);
+#endif /* CONFIG_CLOCK_STM32_PLL2_Q_ENABLE */
+#if defined(CONFIG_CLOCK_STM32_PLL2_R_ENABLE)
+	LL_RCC_PLL2R_Enable();
+	LL_RCC_PLL2_SetR(CONFIG_CLOCK_STM32_PLL2_R_DIVISOR);
+#endif /* CONFIG_CLOCK_STM32_PLL2_R_ENABLE */
+	LL_RCC_PLL2_Enable();
+	while (LL_RCC_PLL2_IsReady() != 1) {
+	}
+#endif /* CONFIG_CLOCK_STM32_PLL2_ENABLE */
```

Comment:
```suggestion
	}
#else
	/* PLL will stay in reset state configuration */
#endif /* CONFIG_CLOCK_STM32_PLL2_ENABLE */
```

`#else` is not necessary, bur provides same way definition as for PLL1 then.

Reply:
Done